blips/blip-3-peerreview.md

```diff
@@ -0,0 +1,53 @@
+This BLIP is a draft: the collaborative document we use to write it is https://hackmd.io/QtoxlUimQNSJByIRKm0IwQ?edit
+
+# Peer Review
+
+| BLIP:     | 3                                                        |
+| -------- | ------------------------------------------------------------ |
+| Title:   | Peer Review                                      |
+| Owner(s):  |                                                            |
+| Author(s):  | Antonio Tenorio Fornés (antonio@decentralized.science), Kaan Uzdogan (uzdogan@mpdl.mpg.de), Paulo Colombo (pcolombo@ucm.es), Elena Pérez Tirador (elena@decentralized.science)                                                           |
+| Status:  | ![Draft](http://rfc.unprotocols.org/spec:2/COSS/draft.svg) |
+| Created: | 2020-06-09                                                   |
+| License: | BSD-2-Clause                                                 |
+
+A draft for a standardized peer review format
+
+## Standard Peer Review registries
+Currently, peer-review-app is registering many fields of a peer review such as author of the review, or the ID of the reviewed article.
+
+This standard should specify which fields to register in every record, the format and standards those fields should adopt, which are mandatory among others.
+
+
+### Peer Review Records' fields
+> [name=Paulo] What is the `id` field and why do we need it ?
+
+| Field | data type | Format/s | Mandatory |  Comments|
+| -------- | ---- | ---- | -------- |----|
+|     id | string  | IPFS Hash??   | Yes     | The id should be unique. How to make sure reviews are unique? See [Zooko's triangle](https://en.wikipedia.org/wiki/Zooko%27s_triangle) for a relevant discussion on the compromises between decentralization, human usability and unique identifiers   |
```

Comment:
Thanks @kiview, this is an interesting discussion to have. From my point of view, there should not be duplicate reviews in the system, meaning that a given review, by a given author for a given manuscript should be only included once in the system. Ideally, the identifier should be unique for that purpose. The hash of the first transaction referring to that id could be indeed a great candidate. If we want to allow updates, an IPFS address of other hashes are not ideal identifiers.

This also opens a new discussion, how to deal with duplicate reviews in the system?

Reply:
I think we can conceptually differentiate between two things:

1. Having 1 logical instance of a peer-review. 
2. Having multiple physical instances of a peer-review record saved on the ledger. 

Multiple records on the ledger can very well reflect (auditable) updates to the record, so this is of course fine. However, how can we ensure that a bad actor isn't publishing multiple instances of the same logical peer-review, with each getting a new unique id? Do we need a locality sensitive hash for this? 

We are in touch with the folks from https://iscc.codes/ for another BLIP, maybe this approach can help here as well (very similar use case actually).

Reply:
I don't think hashes can help us there. Maybe we need some ways to bring trust to the provenance of the peer review record. There is already a mechanism to do it with vouchers. We could promote trustworthy actors to vouche for reviews, such as Journals, maybe authors, and maybe other review registries (Orcid, Publons, F1000, Crossref, etc).

As an advance strategy, we could use something like a Token Curated Registry to vouch for identities, Only trusted identities will have strong weight when vouching, and they could loose trust if they misbehave.

Reply:
@atfornes Agreed - I don't believe hashes can solve the problem of bad actors in this case. The preliminary voucher (Endorser :P) system we developed is quite simplistic, needing only a single other person to endorse the review in order for it to become valid. I like the system in general because it takes advantage of the decentralization of the blockchain, but also social networking effects to hopefully make the app more attractive and interactive. The caveat is that it needs a more robust implementation in its current state (weighting mechanism, 'trusted' endorser, etc).